File: ANSv4/Catalogo3/classes/ReciboItem.vb
Option Explicit On

Public Class ReciboItem


    ' Define como se llama este modulo para el control de errores
    Private Const m_sMODULENAME_ As String = "clsReciboItem"

    'variables locales para almacenar los valores de las propiedades
    Private mvarTipoValor As Byte
    Private mvarImporte As Single
    Private mvarT_Cambio As Single
    Private mvarF_EmiCheque As Date
    Private mvarF_CobroCheque As Date
    Private mvarN_Cheque As String
    Private mvarNrodeCuenta As String
    Private mvarBanco As Integer
    Private mvarCpa As String
    Private mvarChequePropio As Boolean

    Public Property TipoValor() As Byte
        Set(ByVal value As Byte)
            mvarTipoValor = value
        End Set
        Get
            Return mvarTipoValor
        End Get
    End Property

    Public Property Importe() As Single
        Set(ByVal value As Single)
            mvarImporte = value
        End Set
        Get
            Return mvarImporte
        End Get
    End Property

    Public Property T_Cambio() As Single
        Get
            Return mvarT_Cambio
        End Get

        Set(ByVal value As Single)
            mvarT_Cambio = value
        End Set
    End Property

    Public Property F_EmiCheque() As Date
        Set(ByVal value As Date)
            mvarF_EmiCheque = value
        End Set
        Get
            Return mvarF_EmiCheque
        End Get
    End Property

    Public Property F_CobroCheque() As Date
        Set(ByVal value As Date)
            mvarF_CobroCheque = value
        End Set
        Get
            Return mvarF_CobroCheque
        End Get
    End Property

    Public Property N_Cheque() As String
        Set(ByVal value As String)
            mvarN_Cheque = value
        End Set
        Get
            Return mvarN_Cheque
        End Get
    End Property

    Public Property NrodeCuenta() As String
        Set(ByVal value As String)
            mvarNrodeCuenta = value
        End Set

        Get
            Return mvarNrodeCuenta
        End Get
    End Property

    Public Property Banco() As Integer
        Set(ByVal value As Integer)
            mvarBanco = value
        End Set
        Get
            Return mvarBanco
        End Get
    End Property

    Public Property Cpa() As String
        Set(ByVal value As String)
            mvarCpa = value
        End Set
        Get
            Return mvarCpa
        End Get
    End Property

    Public Property ChequePropio() As Boolean
        Set(ByVal value As Boolean)
            mvarChequePropio = value
        End Set
        Get
            Return mvarChequePropio
        End Get
    End Property


End Class

File: ANSv4/Catalogo4/_recibos/ReciboItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Catalogo._recibos
{
    public class ReciboItem
    {

        // Define como se llama este modulo para el control de errores

        //variables locales para almacenar los valores de las propiedades
        private byte mvarTipoValor;
        private float mvarImporte;
        private float mvarT_Cambio;
        private System.DateTime mvarF_EmiCheque;
        private System.DateTime mvarF_CobroCheque;
        private string mvarN_Cheque;
        private string mvarNrodeCuenta;
        private int mvarBanco;
        private string mvarCpa;

        private bool mvarChequePropio;
        public byte TipoValor
        {
            get { return mvarTipoValor; }
            set { mvarTipoValor = value; }
        }

        public float Importe
        {
            get { return mvarImporte; }
            set { mvarImporte = value; }
        }

        public float T_Cambio
        {
            get { return mvarT_Cambio; }
            set { mvarT_Cambio = value; }
        }

        public System.DateTime F_EmiCheque
        {
            get { return mvarF_EmiCheque; }
            set { mvarF_EmiCheque = value; }
        }

        public System.DateTime F_CobroCheque
        {
            get { return mvarF_CobroCheque; }
            set { mvarF_CobroCheque = value; }
        }

        public string N_Cheque
        {
            get { return mvarN_Cheque; }
            set { mvarN_Cheque = value; }
        }

        public string NrodeCuenta
        {

            get { return mvarNrodeCuenta; }
            set { mvarNrodeCuenta = value; }
        }

        public int Banco
        {
            get { return mvarBanco; }
            set { mvarBanco = value; }
        }

        public string Cpa
        {
            get { return mvarCpa; }
            set { mvarCpa = value; }
        }

        public bool ChequePropio
        {
            get { return mvarChequePropio; }
            set { mvarChequePropio = value; }
        }
    }
}
